accounting/payables/supplier_bills/ocr.rst

```diff
@@ -0,0 +1,39 @@
+=====================================================================
+Digitalize your Vendor Bills with Optical Character Recognition (OCR)
+=====================================================================
+
+Encoding your bills manually can be a time-consuming task. Being able to digitalize them and
+extract the data into your database automatically will allow you to reduce errors, save time and
+have an effortless process.
+
+Set up the feature
+==================
+
+Go to :menuselection:`Accounting --> Settings --> Bill Digitalization`, and choose whether the bills
+should be processed automatically or manually.
+
+.. image:: media/setup_ocr.png
+   :align: center
+   :alt: Activate the feature going to the settings application in Odoo Accounting
+
+Start digitalizing your bills
```

Comment:
Digitizing 

Obviously when it's part of the software leave it but fix titles/sentences :)